x/tx/signing/aminojson/encoder.go

```diff
@@ -166,34 +166,29 @@ func moduleAccountEncoder(_ *Encoder, msg protoreflect.Message, w io.Writer) err
 // also see:
 // https://github.com/cosmos/cosmos-sdk/blob/b49f948b36bc991db5be431607b475633aed697e/proto/cosmos/crypto/multisig/keys.proto#L15/
 func thresholdStringEncoder(enc *Encoder, msg protoreflect.Message, w io.Writer) error {
-	pk := &multisig.LegacyAminoPubKey{}
-	msgDesc := msg.Descriptor()
-	fields := msgDesc.Fields()
-	if msgDesc.FullName() != pk.ProtoReflect().Descriptor().FullName() {
+	pk, ok := msg.Interface().(*multisig.LegacyAminoPubKey)
```

Comment:
still need threshold fix right?

Reply:
which one?

Reply:
the fix is not in upstream?

Reply:
[seems](https://github.com/cosmos/cosmos-sdk/pull/22088/files) is not merged yet due to test conflicts, do we wait for it?

Reply:
yeah, I marked this PR as draft